integration-tests-scala/src/test/scala/tech/beshu/ror/integration/plugin/ClusterStateWithInternodeSslTests.scala

```diff
@@ -18,22 +18,31 @@ package tech.beshu.ror.integration.plugin
 
 import com.dimafeng.testcontainers.ForAllTestContainer
 import org.scalatest.Matchers._
-import org.scalatest.WordSpec
+import org.scalatest.{BeforeAndAfterAll, WordSpec}
 import tech.beshu.ror.utils.containers.ReadonlyRestEsCluster.AdditionalClusterSettings
 import tech.beshu.ror.utils.containers.{ReadonlyRestEsCluster, ReadonlyRestEsClusterContainer}
 import tech.beshu.ror.utils.elasticsearch.ClusterStateManager
 
-class ClusterStateWithInternodeSslTests extends WordSpec with ForAllTestContainer {
+import scala.collection.JavaConverters._
 
-  override lazy val container: ReadonlyRestEsClusterContainer = ReadonlyRestEsCluster.createLocalClusterContainer(
-    name = "ROR1",
-    rorConfigFileName = "/cluster_state_internode_ssl/readonlyrest.yml",
-    clusterSettings = AdditionalClusterSettings(
-      numberOfInstances = 3,
-      internodeSslEnabled = true
-    )
-  )
+class ClusterStateWithInternodeSslTests
+  extends WordSpec
+    with ForAllTestContainer
+    with BeforeAndAfterAll {
 
+  override lazy val container: ReadonlyRestEsClusterContainer = {
+    ReadonlyRestEsCluster.createLocalClusterContainer(
+      name = "ROR1",
+      rorConfigFileName = "/cluster_state_internode_ssl/readonlyrest.yml",
+      clusterSettings = AdditionalClusterSettings(
+        numberOfInstances = 3,
+        internodeSslEnabled = true
+      )
+    ).mapContainer { c =>
```

Comment:
I'd rather add new field to `AdditionalClusterSettings`, `envs: Map[String, String]` - we don't need to expose raw container, because the java api is not so clean and simply it looks poor.